From 6ea0efcced33afdf306a03d1b556650a205e8e14 Mon Sep 17 00:00:00 2001
From: rusty-snake <41237666+rusty-snake@users.noreply.github.com>
Date: Wed, 2 Aug 2023 18:55:11 +0200
Subject: [PATCH 2/2] Fix exception when calling bubblejail without arguments

---
 src/bubblejail/bubblejail_cli.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/bubblejail/bubblejail_cli.py b/src/bubblejail/bubblejail_cli.py
index 7d7439f..8d3db86 100644
--- a/src/bubblejail/bubblejail_cli.py
+++ b/src/bubblejail/bubblejail_cli.py
@@ -219,7 +219,7 @@ def create_arg_parser() -> ArgumentParser:
 
 def bubblejail_main(arg_list: Optional[list[str]] = None) -> None:
     # Short circuit to auto-complete
-    if argv[1] == 'auto-complete':
+    if len(argv) > 1 and argv[1] == 'auto-complete':
         from .bubblejail_cli_autocomplete import run_autocomplete
         run_autocomplete()
         return
-- 
2.41.0

